### PR TITLE
Added ability to flush events

### PR DESCRIPTION
**Pull request by botherder**

### CODE REVIEW

**Comment by botherder** [comment]
Anything blocking the merge of this?

File: blockade/libs/events.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """Events interface for blockade."""
 
-__author__ = 'Brandon Dixon'
+__author__ = 'Claudio Guarnieri'
```
Comment (9b) [inline blockade/libs/events.py]: Har. Har. I will let you keep this.

**Comment by 9b** [comment]
Sorry, I had just missed this before.

File: blockade/libs/events.py (hunk 1 above)
Comment (botherder) [inline blockade/libs/events.py]: Remove if you like, but given I added the whole file... ;)

**Pull request merged**